munuSSM fails to compile

In the nightly test the following compilation error occurred:
~~~
models/munuSSM/munuSSM_l_to_lgamma.cpp: In function ‘double flexiblesusy::munuSSM_l_to_lgamma::lepton_total_decay_width(const T1&, const T2&, const flexiblesusy::munuSSM_mass_eigenstates&, const softsusy::QedQcd&)’:
models/munuSSM/munuSSM_l_to_lgamma.cpp:54:43: error: ‘Fe’ was not declared in this scope
    const auto leptonInMass = context.mass<Fe>(indices1);
                                           ^~
models/munuSSM/munuSSM_l_to_lgamma.cpp:54:43: note: suggested alternative: ‘Re’
    const auto leptonInMass = context.mass<Fe>(indices1);
                                           ^~
                                           Re
models/munuSSM/munuSSM_l_to_lgamma.cpp:58:29: error: ‘Fe’ is not a class, namespace, or enumeration
    const double Alpha = Sqr(Fe::electric_charge * unit_charge(context))/(4*Pi);
                             ^~
~~~
The error comes from within the `*_l_to_lgamma.cpp` module.

@wkotlarski would you mind having a look what's going wrong there?

--- a/meta/BrLToLGamma.m
+++ b/meta/BrLToLGamma.m
@@ -106,7 +106,9 @@ CreateInterfaceFunctionForBrLToLGamma[inFermion_ -> {outFermion_, spectator_}] :
                   "// eq. 51 of arXiv:hep-ph/9510309 (note that we include 'e' in the definition of form_factor)\n" <>
                   "const double partial_width = pow(leptonInMassOS,5)/(16.0*Pi) * (std::norm(form_factors[2]) + std::norm(form_factors[3]));\n" <>
 
-                  "const double total_width = lepton_total_decay_width(indices1, indices2, model, qedqcd);\n" <>
+                  "const double total_width = lepton_total_decay_width<" <>
+                     CXXNameOfField[inFermion] <> ", " <> CXXNameOfField[outFermion] <> 
+                     ">(indices1, indices2, model, qedqcd);\n" <>
                   "\nreturn partial_width/total_width;\n"
                ] <> "}";
 

--- a/templates/l_to_lgamma.cpp.in
+++ b/templates/l_to_lgamma.cpp.in
@@ -44,18 +44,18 @@ using namespace @ModelName@_FFV_form_factors;
 namespace flexiblesusy {
 namespace @ModelName@_l_to_lgamma {
 
-template <typename T1, typename T2>
+template <typename FIn, typename FOut, typename T1, typename T2>
 double lepton_total_decay_width(
       T1 const& indices1, T2 const& indices2, 
       const @ModelName@_mass_eigenstates& model, const softsusy::QedQcd& qedqcd) {
 
    context_base context {model};
 
-   const auto leptonInMass = context.mass<Fe>(indices1);
-   const auto leptonOutMass = context.mass<Fe>(indices2);
+   const auto leptonInMass = context.mass<FIn>(indices1);
+   const auto leptonOutMass = context.mass<FOut>(indices2);
    const auto r = pow(leptonOutMass/leptonInMass, 2);
    const auto GF2 = pow(qedqcd.displayFermiConstant(), 2);
-   const double Alpha = Sqr(Fe::electric_charge * unit_charge(context))/(4*Pi);
+   const double Alpha = Sqr(FIn::electric_charge * unit_charge(context))/(4*Pi);
 
    // eq. 10.6 of http://pdg.lbl.gov/2018/reviews/rpp2018-rev-standard-model.pdf
    return 

